node_modules/nodebb-theme-harmony/templates/partials/header/brand.tpl

```diff
@@ -13,23 +14,85 @@
             </a>
             {{{ end }}}
         </div>
-        <!-- Add the links next to the logo -->
-        <div class="d-flex align-items-center gap-3">
-            <!-- Render links for each category -->
-            <div class="nav-item mx-2">
-                <a class="text-reset" href="{config.relative_path}/category/1/announcements" itemprop="url">Announcements</a>
-            </div>
-            <div class="nav-item mx-2">
-                <a class="text-reset" href="{config.relative_path}/category/2/general-discussion" itemprop="url">General Discussion</a>
-            </div>
-            <div class="nav-item mx-2">
-                <a class="text-reset" href="{config.relative_path}/category/4/comments-feedback" itemprop="url">Comments & Feedback</a>
-            </div>
-            <div class="nav-item mx-2">
-                <a class="text-reset" href="{config.relative_path}/category/3/blogs" itemprop="url">Blogs</a>
-            </div>
-            
-        </div>
+
+        <!-- Conditionally render the links next to the logo based on URL -->
+        <script>
+            let debounceTimeout;
+
+            function updateLinks() {
+                var currentUrl = window.location.pathname;
+                console.log("CURRENT URL IS ", currentUrl)
+
+                // Clear existing links
+                var existingLinksContainer = document.querySelector('.dynamic-links-container');
+                if (existingLinksContainer) {
+                    existingLinksContainer.remove();
+                }
+
+                var linksContainer = document.createElement('div');
+                linksContainer.className = 'd-flex align-items-center gap-3 dynamic-links-container';
+
+                // Define categories and their respective URLs
+                var categories = [
+                    { url: '{config.relative_path}/category/1/announcements', text: 'Announcements' },
```

Comment:
The header should not exist on the main home page since the category links are technically present on the page already. I will work on this to achieve full user story completion.